### ISSUE TITLE
Add Graphormer mod

### ISSUE THREAD

**Issue by stsouko** [pytorch-labs/attention-gym#8]
In graphormer bias to attention scores are learnable.

https://github.com/microsoft/Graphormer

**Comment by drisspg** [comment]
Yup, we want to support trainable biases. There is some subtlety with how to implement this -> Broadcasts in the forward become reductions in the backwards. We are investigating though the best way to enable this

**Comment by jozhang97** [comment]
Following, also interested in learnable pair bias

**Comment by drisspg** [comment]
Added a represntative example: https://github.com/meta-pytorch/attention-gym/pull/218
